### PR TITLE
Add gii

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

- [x] Added Gii support
- [x] Raised PHP version up to 8.1

### CODE REVIEW

**Comment by samdark** [comment]
Ready for review?

**Comment by xepozz** [comment]
Yeap, but actually it doesn't work because of `config/params-web.php`. If I leave the settings from the `config/params-web.php` in the `params.php` it will break some `yii-gii` processes.

**Comment by rustamwin** [comment]
FYI, I'll remove `view` dependency from Gii soon.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
     name: PHP ${{ matrix.php }}-${{ matrix.os }}
 
     env:
-      extensions: fileinfo, pdo, pdo_sqlite, intl
+      extensions: fileinfo, pdo, pdo_sqlite, intl-70.1
```
Comment (samdark) [inline .github/workflows/build.yml:34]: Why 70.1?

File: tests/HttpClient/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.private.env.json
```
Comment (samdark) [inline tests/HttpClient/.gitignore:1]: Not sure we need that.

File: .github/workflows/build.yml (hunk 1 above)
Comment (xepozz) [reply .github/workflows/build.yml:34]: Windows suddenly failed without it

**Review by samdark — APPROVED** [review]

**Pull request merged**